src/search/components/SearchResultItem.tsx

```diff
@@ -64,14 +65,38 @@ const SearchResultItem: FunctionComponent<SearchResultItemProps> = ({
 			/>
 		);
 	};
+	const checkIfResultIsNotAnItem = (result: Avo.Search.ResultItem): boolean => {
+		return result.administrative_type === ('collectie' || 'bundel' || 'opdracht');
+	};
+
+	const renderAuthorOrOrganisation = (result: Avo.Search.ResultItem) => {
+		if (checkIfResultIsNotAnItem(result)) {
+			const name = `${result.owner?.firstname} ${result.owner?.lastname}`;
+			const initials = `${result.owner?.firstname?.[0]}${result.owner?.lastname?.[0]}`;
+
+			return (
+				<Avatar
+					image={result.owner?.avatar_path || undefined}
+					name={name}
+					initials={initials.toLocaleUpperCase()}
+					size="small"
+					dark
+				/>
+			);
+		}
+
+		if (result.original_cp) {
+			return renderSearchLink(
+				result.original_cp,
+				{ filters: { provider: [result.original_cp] } },
+				'c-body-2'
+			);
+		}
+	};
```

Comment:
What happens if both if statements are false
We should return something, even if it is just: return null;